src/bot.ts

```diff
@@ -17,6 +17,7 @@ export let eventHandlers: EventHandlers = {};
 export let proxyWSURL = `wss://gateway.discord.gg`;
 
 export async function startBot(config: BotConfig) {
+  if (!config.token) throw Error("No token provided!")
```

Comment:
Good Idea but please add the error to our Errors enum and don't forget the new keyword so like
`throw new Error(Errors.NO_TOKEN_PROVIDED)`